packages/manager/apps/dedicated/client/app/dedicatedCloud/datacenter/drp/configuration/ovh/secondPcc/dedicatedCloud-datacenter-drp-ovh-secondPccStep.controller.js

```diff
@@ -126,34 +126,47 @@ export default class {
         this.drpInformations.primaryPcc.generation !== this.PCC_NEW_GENERATION,
       )
       .then((enableDrp) => {
+        let storeOptionPromise = Promise.resolve();
         if (enableDrp.url !== undefined) {
-          this.storeZertoOptionOrderInUserPref(this.drpInformations, enableDrp);
+          storeOptionPromise = this.storeZertoOptionOrderInUserPref(
+            this.drpInformations,
+            enableDrp,
+          );
+
           if (!enableDrp.hasAutoPay) {
             this.$window.open(enableDrp.url, '_blank');
           }
         }
 
-        return this.goToPccDashboard(true).then(() => {
-          // $timeout necessary to display alerter message
-          this.$timeout(() => {
-            if (!enableDrp.hasAutoPay) {
-              this.displaySuccessMessage(
-                `${this.$translate.instant(
-                  'dedicatedCloud_datacenter_drp_confirm_order',
-                  { billUrl: enableDrp.url },
-                )}`,
-              );
-            } else {
-              this.displayInfoMessage(`
+        return Promise.all([
+          storeOptionPromise,
+          this.setDisableSuccessAlertPreference(
+            this.drpInformations.primaryPcc.serviceName,
+            false,
+          ),
+        ])
+          .then(() => this.goToPccDashboard(true))
+          .then(() => {
+            // $timeout necessary to display alerter message
+            this.$timeout(() => {
```

Comment:
It wouldn't be if you used `$q` instead, wouldn't it ? 🤔

Reply:
Hmm this was done because of the scope of `dedicatedCloud` controller and template, but it seems now useless => to remove